### DISCUSSION TITLE
S3 gateway: hide certain buckets

### DISCUSSION THREAD

**Discussion by schwichti** [minio/minio#13309] [Q&A]
Hi,
I am running MinIO version `2020-09-17T04:49:20Z` in S3 gateway mode. I want to configure MinIO in a way that the MinIO user can list only one specific bucket, while all other buckets are hidden. I thought that this should be possible using a proper IAM policy. I started with the policy posted here: https://docs.min.io/docs/minio-gateway-for-s3.html. The best I can do is to deny access to other buckets, but they are still listed. I tried so many policies that I question if that is even possible.

**Comment by harshavardhana** [comment]
> Hi,
> I am running MinIO version `2020-09-17T04:49:20Z` in S3 gateway mode. I want to configure MinIO in a way that the MinIO user can list only one specific bucket, while all other buckets are hidden. I thought that this should be possible using a proper IAM policy. I started with the policy posted here: https://docs.min.io/docs/minio-gateway-for-s3.html. The best I can do is to deny access to other buckets, but they are still listed. I tried so many policies that I question if that is even possible.

That is not the feature AWS S3 supports.
